<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GUI_AudioConverter</class>
 <widget class="QWidget" name="GUI_AudioConverter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>225</width>
    <height>144</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="verticalSpacing">
    <number>3</number>
   </property>
   <item row="10" column="0" colspan="2">
    <widget class="QStackedWidget" name="sw_progress">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="page_4">
      <layout class="QGridLayout" name="gridLayout_5">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="1">
        <widget class="QPushButton" name="btn_start">
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_5">
      <layout class="QGridLayout" name="gridLayout_4">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QProgressBar" name="pb_progress">
         <property name="value">
          <number>24</number>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="btn_stop_encoding">
         <property name="text">
          <string>Stop</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QSpinBox" name="sb_threads">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>32</number>
     </property>
     <property name="value">
      <number>4</number>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="lab_threads">
     <property name="text">
      <string>#Threads</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QStackedWidget" name="sw_preferences">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="3">
        <widget class="QSpinBox" name="sb_ogg_quality">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>10</number>
         </property>
         <property name="value">
          <number>7</number>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Quality</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="1">
        <widget class="Gui::ComboBox" name="combo_cbr">
         <property name="currentText">
          <string notr="true">320</string>
         </property>
         <item>
          <property name="text">
           <string notr="true">320</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">256</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">192</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">160</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">128</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">96</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Quality</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_3">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Quality</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="sb_lame_vbr">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>10</number>
         </property>
         <property name="value">
          <number>7</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="Gui::ComboBox" name="combo_codecs">
     <property name="currentText">
      <string notr="true">Ogg Vorbis</string>
     </property>
     <item>
      <property name="text">
       <string notr="true">Ogg Vorbis</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">MP3 (CBR)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">MP3 (VBR)</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="9" column="0" colspan="2">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Gui::ComboBox</class>
   <extends>QComboBox</extends>
   <header>Gui/Utils/Widgets/ComboBox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>combo_cbr</tabstop>
  <tabstop>sb_threads</tabstop>
  <tabstop>btn_stop_encoding</tabstop>
  <tabstop>btn_start</tabstop>
  <tabstop>sb_ogg_quality</tabstop>
  <tabstop>combo_codecs</tabstop>
  <tabstop>sb_lame_vbr</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
